examples/fewshot_protonet/configs/template.yaml

```diff
@@ -0,0 +1,28 @@
+# This is a demo config file for protonet.
```

Comment:
Consider using a different starting description to differentiate it from demo.yaml

Reply:
Changed to a detail description